crates/dbsp/src/circuit/dbsp_handle.rs

```diff
@@ -885,7 +885,6 @@ impl DBSPHandle {
     fn send_complete_bootstrap(&mut self) -> Result<(), DbspError> {
         self.broadcast_command(Command::CompleteBootstrap, |_, _| {})?;
 
-        info!("Boostrap complete");
```

Comment:
this seemed useful

Reply:
it was a duplicate, it's already logged elsewhere